libs/sdk-ui-filters/src/AttributeFilterHandler/internal/redux/loadAttribute/loadAttributeSaga.ts

```diff
@@ -49,7 +50,13 @@ export function* loadAttributeSaga(
             displayFormRef,
         );
 
-        yield put(actions.loadAttributeSuccess({ attribute, correlation }));
+        const attributeDataSet: PromiseFnReturnType<typeof loadAttributeDataSetMeta> = yield call(
```

Comment:
Should we make this call optional based on `shouldDisplayAttributeTooltip` ?

Reply:
There's currently plans to extend Attribute Filter list with also these information - [TNT-1387](https://gooddata.atlassian.net/browse/TNT-1387)
So I think, having it under `shouldDisplayAttributeTooltip` will not last long.